### DISCUSSION TITLE
how i find location with city nam whithout lat an long

### DISCUSSION THREAD

**Discussion by MasterDev001** [open-meteo/open-meteo#451] [General]
how i find location with city nam whithout lat an long

**Comment by patrick-zippenfenig** [comment]
You can use the Geocoding API to find coordinates for cities: https://open-meteo.com/en/docs/geocoding-api

It is not possible to call the weather API directly with a city name, as this name could be ambiguous. Using coordinates is the most accurate and scalable solution.

**Reply by MasterDev001** [reply]
Thank you but how do I find a city name with latitude, and longitude

**Reply by MasterDev001** [reply]
can i find city info with this link https://open-meteo.com/en/docs/geocoding-api

**Reply by Wemaka** [reply]
use [this](https://www.geonames.org/export/web-services.html) to get the settlement by longitude and latitude
